recipes/ca-certificates/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2016.2.28" %}
+
+package:
+  name: ca-certificates
+  version: {{ version }}
+
+# This does not have a source because it pulls these from certifi at present.
+
+build:
+  number: 0
+  skip: true  # [win or not py35]
+  # As openssl includes these (dependency of Python), they will be removed afterwards.
+  # This is our way of ensuring these files are kept anyways. Once conda-forge has its
+  # own openssl, this will no longer be necessary.
+  always_include_files:
+    - ssl/cert.pem
+    - ssl/cacert.pem
+
+requirements:
+  build:
+    - python
+    - certifi {{ version }}
+
+test:
+  requires:
+    - curl
+
+  commands:
+    # Verify the certificates are there.
+    - test -f "${PREFIX}/ssl/cacert.pem"  # [unix]
+    - test -f "${PREFIX}/ssl/cert.pem"    # [unix]
+
+    # Use the certificates to download something.
+    - curl --cacert "${PREFIX}/ssl/cacert.pem" https://www.google.com
```

Comment:
Not only do we ensure the certificates are in place, but we use them to do something.